### PR TITLE
VIR-91 Employee modal

### PR DESCRIPTION
**Pull request by nniemi**
Added modal to add employee to the position
Also added frontend implementation for editing the employee to make editing smoother

### CODE REVIEW

**Review by joonasyr — DISMISSED** [review]
Seems good to me

**Review by joonasyr — DISMISSED** [review]
👍

File: backend/Virkarekisteri/src/Functions/Employees/GetPositionEmployee.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+)
+{
+    /// <summary>
+    /// Gets a position employee by id
+    /// </summary>
+    /// <param name="req">Input GET request containing the ID</param>
+    /// <returns>
+    /// The position employee with the given ID
+    /// </returns>
+    [Function("GetPositionEmployee")]
```
Comment (0x464e) [inline backend/Virkarekisteri/src/Functions/Employees/GetPositionEmployee.cs:22]: You need to add the required authorization attribute here

**Review by 0x464e — CHANGES_REQUESTED** [review]

File: backend/Virkarekisteri/src/Functions/Employees/GetPositionEmployee.cs (hunk 1 above)
Comment (nniemi) [reply backend/Virkarekisteri/src/Functions/Employees/GetPositionEmployee.cs:22]: done

**Pull request merged**